pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -93,16 +93,18 @@ stdenv.mkDerivation ({
 
   patches = [
     ./env_var_for_system_dir.patch
-  ] ++ lib.optionals (stdenv.isAarch64) [
-    (fetchpatch {
-      url = "https://raw.githubusercontent.com/archlinuxarm/PKGBUILDs/09c7fa0dc1d87922e3b464c0fa084df1227fca79/extra/firefox/arm.patch";
-      sha256 = "1vbpih23imhv5r3g21m3m541z08n9n9j1nvmqax76bmyhn7mxp32";
-    })
+  ] ++ lib.optionals (stdenv.isAarch64) ([
```

Comment:
I was thinking: Does it harm if we always apply that patch? It would certainly mean we notice it earlier if it fails to apply.

Reply:
It shouldn't [h]arm in most cases, but I haven't reviewed the opus patch to know exactly what it does.

And you're right, I would think it's preferable to unconditionally patch, if the patch has issues on other platforms, it means it's not upstreamable as-is, and we probably shouldn't include patches that are not upstreamable.

Reply:
I couldn't find an upstream issue for that opus patch. I'd rather not include it by default then. The other patch (https://github.com/NixOS/nixpkgs/pull/81211/files#diff-b39b9a678f9ecf5d0e1581832f7bf99dR104) was merged upstream and should probably just be included at all times.

Reply:
I just build successfully without opus patch.
upstream info:
https://bugzilla.mozilla.org/show_bug.cgi?id=1530659

Reply:
Nice! @samueldr can you remove the opus patch and then merge this?